share/spice/dogo_movies/dogo_movies.js

```diff
@@ -0,0 +1,60 @@
+(function(env) {
+    "use strict";
+
+    env.ddg_spice_dogo_movies = function(api_result) {
+        if (!api_result || !api_result.results || !api_result.results.length) {
+            return Spice.failed('dogo_movies');
+        }
+
+        // Get original query.
+        var script = $('[src*="/js/spice/dogo_movies/"]')[0],
+            source = $(script).attr("src"),
+            query = decodeURIComponent(source.match(/dogo_movies\/([^\/]+)/)[1]);
+
+        Spice.add({
+            id: 'dogo_movies',
+            name: 'Kids Movies',
+            data: api_result.results,
+            meta: {
+                sourceName: 'DOGOmovies',
+                sourceUrl: 'http://www.dogomovies.com/search?query=' + encodeURIComponent(query) + '&ref=ddg',
+                itemType: 'kids movies'
+            },
+            normalize: function(item) {
+                var thumb = item.thumb;
+                return {
+                    title: item.name,
+                    image: thumb,
+                    img: thumb,
+                    img_m: thumb,
+                    heading: item.name,
+                    rating: item.ratings.score,
+                    ratingText: item.comments_count + ' reviews',
+                    reviewCount: item.comments_count,
+                    url: item.url,
+                    abstract: Handlebars.helpers.ellipsis(item.summary, 200)
+                };
+            },
+            templates: {
+                group: 'media',
+                options: {
+                    variant: 'poster',
+                    buy: Spice.dogo_movies.buy,
+                    subtitle_content: Spice.dogo_movies.subtitle_content,
```

Comment:
If you add `rating: true` you'll see the stars display below the image in the tile view:

![2015-04-01_17h38_44](https://cloud.githubusercontent.com/assets/873785/6953233/fd68a194-d895-11e4-8f2d-2358c2123250.png)